base/pom.xml

```diff
@@ -192,6 +192,29 @@ of Jackson: application code should only rely on `jackson-bom`
           </configuration>
 	</plugin>
 
+        <plugin>
+          <groupId>org.gradlex</groupId>
+          <artifactId>gradle-module-metadata-maven-plugin</artifactId>
+          <version>1.0</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>gmm</goal>
+              </goals>
+            </execution>
+          </executions>
+          <configuration>
+            <platformDependencies>
+              <dependency>
```

Comment:
Could use some explanation of what these mean (if explainable in a brief comment)?